### DISCUSSION TITLE
Add. open dislexic fonts and different colors based on the type of account logged in

### DISCUSSION THREAD

**Discussion by albanobattistella** [learningequality/kolibri#12463] [General]
Hello,
The introduction of fonts for dyslexics would be recommended for those people who have reading difficulties.
The introduction of different colors depending on the type of account I am connected to clearly identifies whether I am admin, teacher or learner.

**Comment by MisRob** [comment]
Thanks for your feedback @albanobattistella, I shared it with my colleague @radinamatic whose expertise is a11y.

**Reply by albanobattistella** [reply]
Thank you!

**Comment by MisRob** [comment]
Reported internally so closing
